SemanticFormsSelect.php

```diff
@@ -25,6 +34,19 @@
 
 define( 'SFS_VERSION', '1.3.0' );
 
+if ( version_compare( $GLOBALS['wgVersion'], '1.27c', '>' ) ) {
```

Comment:
Not really since mediawiki 1.25 extension.json support isent as good as in 1.26 and 1.27. But 1.26 dosent support composer in extension.json whereas it does in 1.27.